config.yaml.defaults

```diff
@@ -433,6 +433,8 @@ image_grayscale_params:
 slack:
   microservice_port: 64100
   expected_url_preamble: "https://hooks.slack.com/"
+  followup_requests:
```

Comment:
Could you point out to me where this entry is used?

Reply:
You are totally right, it works without this.